include/myst/times.h

```diff
@@ -6,8 +6,26 @@
 
 #include <time.h>
 
+#include <myst/clock.h>
+
 long myst_lapsed_nsecs(const struct timespec* t0, const struct timespec* t1);
 
+MYST_INLINE void set_timespec_from_nanos(struct timespec* tp, long nanos)
```

Comment:
Suggest changing the function names to "nanos_to_timespec" and "timespec_to_nanos".

Reply:
Changed.